Developer-guide.ipynb

```diff
@@ -0,0 +1,342 @@
+     "language": "pwsh"
+    },
+    "polyglot_notebook": {
+     "kernelName": "pwsh"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "Invoke-Expression \"$tryDotnet\\build.cmd\"\n",
+    "\n",
+    "Invoke-Expression \"$tryDotnet\\build-js.cmd\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## step 6: Open try dotnet in Visual Studio and set Microsoft.TryDotNet as a startup project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## step 7: Run try dotnet (F5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## step 8: code D:\\microsoft\\dotnet\\try\\src\\microsoft-learn-mock and then open menu Terminal and start a new terminal\n",
+    "     "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## step 9:\n",
+    "\n",
+    "npm ci\n",
+    "\n",
+    "npm run buildProd\n",
+    "\n",
+    "npx http-server"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "http://192.168.1.68:8080/site/DocsHost.html?trydotnetUrl=https://localhost:7061/api/trydotnet.min.js&trydotnetOrigin=https://localhost:7061/\n",
+    "\n",
+    "converted to url:\n",
+    "\n",
+    "http://192.168.1.68:8080/site/DocsHost.html?trydotnetUrl=https%3A%2F%2Flocalhost%3A7061%2Fapi%2Ftrydotnet.min.js&trydotnetOrigin=https%3A%2F%2Flocalhost%3A7061%2F"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Try .NET / Microsoft Learn flow diagram\n",
+    "\n",
+    "The following diagram shows the interaction between the Try .NET service (trydotnet.microsoft.com) and a host page (learn.microsoft.com).\n",
+    "\n",
+    "The `CSharpProjectKernel` is the .NET Interactive backend for Try .NET."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "mermaid"
+    },
+    "polyglot_notebook": {
+     "kernelName": "mermaid"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"mermaidMarkdownContainer\" style=\"background-color:white\">\r\n",
+       "<link rel=\"stylesheet\" href=\"https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css\">\r\n",
+       "<div id=\"18394cd2b1eb4dc0a1a21e3973de6734\"></div>\r\n",
+       "<script type=\"module\">\r\n",
+       "\r\n",
+       "            import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10.1.0/dist/mermaid.esm.min.mjs';\r\n",
+       "            let renderTarget = document.getElementById('18394cd2b1eb4dc0a1a21e3973de6734');\r\n",
+       "            try {\r\n",
+       "                const {svg, bindFunctions} = await mermaid.mermaidAPI.render( \r\n",
+       "                    'mermaid_18394cd2b1eb4dc0a1a21e3973de6734', \r\n",
+       "                    `sequenceDiagram\r\n",
+       "    box rgba(33,66,99, 0.5) Browser\r\n",
```

Comment:
We should leave the sequence diagram out of this notebook and maybe just link to the readme in the repo so that we don't have multiple copies of it. Otherwise they'll get out of sync as we make changes.

Reply:
Done.

Removed the diagram and added a link to it